homeassistant/components/telegram_bot/__init__.py

```diff
@@ -85,6 +87,8 @@
     vol.Required(CONF_ALLOWED_CHAT_IDS):
         vol.All(cv.ensure_list, [vol.Coerce(int)]),
     vol.Optional(ATTR_PARSER, default=PARSER_MD): cv.string,
+    vol.Optional(ATTR_PROXY_URL): cv.string,
```

Comment:
Is there a reason to prefix configuration constants with `ATTR_` instead than `CONF_`?